### ISSUE TITLE
Chart missing data

### ISSUE THREAD

**Issue by Fawwad-Khan** [amcharts/amcharts4#1161]
Hey,
Im using StepLineSeries to generate a chart, I have a rather large dataset, and the chart is not displaying all the day, its only showing partial data.
I think it might be that the library is auto adjusting the chart to look the best but I really need to show all the data regardless of how it looks, is it possible in the current version? 
I have a deadline, would really appreciate a quick fix.

Thank you in advance :)

**Comment by martynasma** [comment]
Any demo for us to look at? Or we're supposed to guess what you have for chart setup and data? ;)

**Comment by Fawwad-Khan** [comment]
I'm sorry, I work in a startup, so its a pretty busy schedule, it slips my mind. I'll try and create a code pen for this and the earlier issue but if i couldnt, I'm using a step line chart, where the data needs to be updated on every form submission, previously I was generating the chart on componentUpdate, and when the data updated I just disposed the chart and regenerated it, but it was obviously unreliable and too resource extensive, seeing as the chart was being generated from scratch everytime. So, I changed the flow, in the new flow, the chart was being generated in componentMount and on update instead of being disposed and recreated, I just over wrote the data, which was faster. But I encountered an issue where the graph wasn't showing the full data sometimes, I have a pretty large data set its like 12 hrs of data with 20 seconds intervals. I think the library adjusts the graph to look best, but I don't want it to look good I just want to show the full data, is there a prop or any way to overwrite this rule?
